tests/next_tests/definitions.py

```diff
@@ -146,6 +149,8 @@ class ProgramFormatterId(_PythonObjectIdMixin, str, enum.Enum):
     (USES_SPARSE_FIELDS_AS_OUTPUT, XFAIL, UNSUPPORTED_MESSAGE),
 ]
 DACE_SKIP_TEST_LIST = COMMON_SKIP_TEST_LIST + [
+    # TODO(tehrengruber): disable all DaCe tests until new version of the backend using GTIR lands
```

Comment:
Shouldn't we mark this with the issue tracking the temporary changes for the ongoing GTIR work? I propose to use a `FIXME` tag instead of a `TODO` to avoid mistakes. Something like: 
```suggestion
    # FIXME(#1582): disable all DaCe tests until new version of the backend using GTIR lands
```